SQL_Assignments/sql_5.sql

```diff
@@ -0,0 +1,45 @@
+uber
+
```

Comment:
Use identity values for auto generated tables

Reply:
ok